lib/custom/src/MShop/Service/Provider/Payment/Payone.php

```diff
@@ -22,6 +22,138 @@ class Payone
 	implements \Aimeos\MShop\Service\Provider\Payment\Iface
 {
 	/**
+	 * Returns the data passed to the Omnipay library
+	 *
+	 * @param \Aimeos\MShop\Order\Item\Base\Iface $base Basket object
+	 * @param $orderid Unique order ID
+	 * @param array $params Request parameter if available
+	 */
+	protected function getData( \Aimeos\MShop\Order\Item\Base\Iface $base, $orderid, array $params )
+	{
+		$urls = $this->getPaymentUrls();
+		$desc = $this->getContext()->getI18n()->dt( 'mshop', 'Order %1$s' );
+		$card = $this->getCardDetails( $base, $params );
+
+		// transfer information about product items, too
+		$lines = [];
+		$i = 0;
+
+		foreach( $base->getProducts() as $product )  {
+			$lines[$i] = new \Omnipay\Payone\Extend\Item([
+				'id' => (string) $product->toArray()['order.base.product.prodcode'],
+				'name' => $product->getName(),
+				'itemType' => 'goods', // Available types: goods, shipping etc.
+				'quantity' => $product->getQuantity(),
+				'price' => $product->getPrice()->getValue(),
+				'vat' => (int) $product->getPrice()->getTaxRate(), // Optional
+			]);
+			$i++;
+
+		}
+
+		// if delivery costs are greater than zero, transfer the delivery service item, too
+		if ($base->getService('delivery')->getPrice()->getCosts() != '0.00') {
+			$deliveryObject = $base->getService('delivery');
+			$lines[$i] = new \Omnipay\Payone\Extend\Item([
+				//'id' => (string) $deliveryObject->getId(),
```

Comment:
Can you remove the commented lines with source code to make scrutinizer-ci happy?